.github/workflows/windows.yml

```diff
@@ -1,14 +1,14 @@
 name: Tests on Windows (VS 2022, Python 3.11)
 on:
   workflow_dispatch:
-#  pull_request:
-#    paths-ignore:
-#      - '**/docs/**'
-#      - 'docs/**'
-#      - '**/**.md'
-#      - '**.md'
-#      - '**/layer_tests_summary/**'
-#      - '**/conformance/**'
+  pull_request:
+    paths-ignore:
+      - '**/docs/**'
+      - 'docs/**'
+      - '**/**.md'
+      - '**.md'
+      - '**/layer_tests_summary/**'
+      - '**/conformance/**'
 #  push:
```

Comment:
This should be uncommented for the workflow to work on master.

Reply:
uncommented